src/hooks/Character.js

```diff
@@ -11,9 +11,10 @@ export const useCharacters = () => {
   };
 
   useEffect(() => {
+    getRickAndMortyPeeps(page);
     mapBro[characterType]()
```

Comment:
you need to pass page into the function here - that's why it's not rerendering when you change the page number

Reply:
Got it, thanks Evan!